README.md

```diff
@@ -5,68 +5,13 @@
 ![Lint](https://github.com/aquarist-labs/s3gw/actions/workflows/lint.yaml/badge.svg)
 ![Build](https://github.com/aquarist-labs/s3gw/actions/workflows/build-environment.yml/badge.svg)
 
-An S3-compatible Gateway based on Ceph RGW, using a non-RADOS backend for
-standalone usage.
+We're developing an easy-to-use Open Source and Cloud Native S3 service for
+Kubernetes.
 
 This project provides the required infrastructure to build a container
 able to run on a kubernetes cluster, providing S3-compatible endpoints to
 applications.
```

Comment:
This paragraph seems to be an old remnant of the s3gw-core repo and doesn't fit particularly well into the current structure.

Reply:
Updated